app/controllers/admin_controller.rb

```diff
@@ -15,7 +15,22 @@ def index
       scope = scope.unpaired
       @title = 'Unpaired Users'
     end
-    scope = scope.fuzzy_search(params[:q]) if params[:q].present?
+
+    name_email = params[:name_email]
+    zip = params[:zip]
+    supported = params[:supported]
+
+    if name_email.present?
+         scope = scope.fuzzy_search(name_email) # Searches all string fields; presumably won't match ZIP
```

Comment:
Stick these on one line if you can

`scope = scope.fuzzy_search(name_email) if name_email.present?`